src/main/java/com/cozyhome/onlineshop/userservice/security/service/TokenBlackListService.java

```diff
@@ -1,7 +1,7 @@
 package com.cozyhome.onlineshop.userservice.security.service;
 
 public interface TokenBlackListService {
-    void saveTokenToBlackList(String jwtToken);
+	
+	void saveTokenToBlackList(String jwtToken);
```

Comment:
there's an extra space

Reply:
Do we really need comments for formatting things?